adoc-mode.el

```diff
@@ -248,6 +257,48 @@ See for example `tempo-template-adoc-title-1'."
                  (const :tag "tempo-snippets" tempo-snippets))
   :group 'adoc)
 
+(defcustom adoc-fontify-code-blocks-natively nil
+  "When non-nil, fontify code in code blocks using the native major mode.
+This only works for code blocks where the language is
+specified where we can automatically determine the appropriate
+mode to use.  The language to mode mapping may be customized by
+setting the variable `adoc-code-lang-modes'."
+  :group 'markdown
```

Comment:
Why a different group?

Reply:
Copy-Paste-Error (CPE). Thanks for indicating it.